cli/test_cmd.go

```diff
@@ -114,6 +114,16 @@ func (c *TestCmd) Execute(args []string) error {
 			}
 			expectedDir := filepath.Join(tree, "../../expected", exeMethod, filepath.Base(tree))
 			actualDir := filepath.Join(tree, "../../actual", exeMethod, filepath.Base(tree))
+
+			_, err := os.Stat(expectedDir)
+			if err != nil && os.IsNotExist(err) {
+				// (alexsaveliev) we may have some tests available only for docker or program mode
```

Comment:
Right, they should behave identical as much as possible, but what Docker image has some pre-requisites missing (e.g. Android SDK)?